lib/init.coffee

```diff
@@ -8,12 +8,19 @@ module.exports =
       type: 'string'
       title: 'Path to the javac executable'
       default: 'javac'
+    classpath:
+      type: 'string'
```

Comment:
I think we should stick with a string.  Java programmers are used to specifying the classpath as a single string.